atc/exec/step_metadata.go

```diff
@@ -69,3 +69,11 @@ func (metadata StepMetadata) Env() []string {
 
 	return env
 }
+
+func (metadata StepMetadata) TaskEnv() []string {
+	env := []string{}
+	if metadata.ExternalURL != "" {
```

Comment:
is there a case that this is not set?

Reply:
I am not sure. But I just copied the piece of existing code:

https://github.com/concourse/concourse/blob/c49d370b904af1950c18b916d0e19fa9ab872d93/atc/exec/step_metadata.go#L62-L64

Reply:
`ExternalURL` will not be set if it wasn't specified here: https://github.com/concourse/concourse/blob/c49d370b904af1950c18b916d0e19fa9ab872d93/atc/atccmd/command.go#L127